app/views/repositories/_staff_list.html.haml

```diff
@@ -3,9 +3,11 @@
   %label.control-label{ for: 'course_course[user_ids][]' }
   = hidden_field_tag "", name: 'course[user_ids][]'
   %label Assign Additional Staff:
-  - staff_members.each do |staff|
-    = render partial: 'shared/formtastic_checkbox', locals: { _klass: 'staff', _for: "course_user_ids_#{staff.id}", _name: 'course[user_ids][]', _id: staff.id, _checked: assigned_staff.map{|assigned| assigned.id}.include?(staff.id), to_display: staff.full_name }
-- else
-  %ul
+  - if staff_members
```

Comment:
I don't believe this is correct - in Ruby, unlike Python, an empty collection is truthy - the only falsey values are nil, false, and anything specifically constructed to be falsey.  And if this is in relation to an undefined error, that error will still occur - you can't check an undefined value for falsiness, only definedness via defined?